backend-api/template.yaml

```diff
@@ -789,6 +789,7 @@ Resources:
           ENCRYPTION_KEY_ARN: !GetAtt KMSEncryptionKey.Arn
           STS_JWKS_ENDPOINT:
             !FindInMap [EnvironmentVariables, !Ref Environment, STSJWKSENDPOINT]
+          STS_JWKS_ENDPOINT_RETRY_DELAY_IN_MS: 1000
```

Comment:
I made comments about `STS_JWKS_ENDPOINT_RETRY_DELAY_IN_MS` being an env var and that was silly me, so I've deleted them. Do we want maxAttempts to be an env var as well?

Reply:
Done